cmd/talosctl/cmd/talos/config.go

```diff
@@ -175,8 +179,50 @@ func openConfigAndContext(context string) (*clientconfig.Config, error) {
 	return c, nil
 }
 
+// configGetContexts represents config contexts command.
+var configGetContexts = &cobra.Command{
+	Use:    "contexts",
+	Short:  "List contexts defined in Talos config",
+	Long:   ``,
+	Hidden: false,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		c, err := clientconfig.Open(Talosconfig)
+		if err != nil {
+			return fmt.Errorf("error reading config: %w", err)
+		}
+
+		keys := make([]string, len(c.Contexts))
+		i := 0
+		for key := range c.Contexts {
+			keys[i] = key
+			i++
+		}
+		sort.Strings(keys)
+
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
+		fmt.Fprintln(w, "CURRENT\tNAME\tENDPOINTS")
```

Comment:
I could see it useful to see the `NODES` as well, but just a suggestion. It could get ugly if they have a lot.

Reply:
initially I've added them, but then had the same though: can be too much. But we can get first N there I guess.